py/desispec/workflow/procfuncs.py

```diff
@@ -779,6 +779,8 @@ def checkfor_and_submit_joint_job(ptable, arcs, flats, sciences, arcjob, flatjob
     elif lasttype == 'arc' and arcjob is None and len(arcs) > 4:
         ptable, arcjob, internal_id = arc_joint_fit(ptable, arcs, internal_id, dry_run=dry_run, queue=queue)
         internal_id += 1
+    if dry_run:
+        time.sleep(1)
```

Comment:
Minor: Consider removing this or making it 0.1 sec.

Reply:
This is only run a handful of times (it's in the joint fit code) and I added it because the mock queue id's are generated using the cpu clock time in seconds. It, therefore, needs to wait at least a second. This prevents two mock submissions from having the same mock Slurm id.

I can move it to within just the arc and flat, where the problem arises. That will reduce the number of total waits in a night, but added more code. So I just went with this.

Reply:
ok, that makes sense.  I thought you were just trying to watch the progress like a movie while debugging, but now I see that it has a specific purpose.  Please add a comment about why it needs to be a second so that someone clever like me doesn't try to remove it in the future.